nnutils/timing/timer.py

```diff
@@ -7,19 +8,29 @@
 class Timer:
     end_time: float = 0.0
     start_time: float = 0.0
-
-    def time_start(self):
+    is_garbage_collector_disabled: bool = False
+    
+    def time_start(self, disable_garbage_collector: bool=True) -> None:
         def _time_start(
             module,
             args,
         ):
+            if disable_garbage_collector:
+                gc.disable()
+                self.is_garbage_collector_disabled = True
             self.start_time = time.time()
+            
 
         return _time_start
 
-    def time_end(self, inference_times: List[float]):
+    def time_end(self, inference_times: List[float]) -> None:
```

Comment:
The output of this function is None but Callable

Reply:
Just updated!